### ISSUE TITLE
Creating Frontend for repometrics.json 

### ISSUE THREAD

**Issue by natalialuzuriaga** [DSACMS/metrics#147]
## MENTOR
- @natalialuzuriaga  
- SWE Fellow, USDigitalCorps
- dm on slack, #wg-government on CHAOSS.community slack, 

## BRIEF DESCRIPTION
WIP

## SCOPING
- 2 weeks

## SKILLS NEEDED
Front-end, Webdev

## REQUIRED DELIVERABLES
- Determine fields for `repometrics.json` with help of Natalia
- Create a design using Figma with fields from this file added to report page
- Using `repometrics.json`, add fields into the project report template

## DESIRED DELIVERY WINDOW
- Any

## NOTES/INSPO
Any feedback and ideas on this issue are welcome!

# ACKNOWLEDGEMENTS
- [X] - Yes, I agree to spend file a project proposal ticket
- [x] - Yes, I agree this project falls within a 2 week or less (ideally less) scope of time
- [x] - Yes, I agree to spend an hour per week of office hours and 1x1 mentoring
- [x] - Yes, I agree to spend an hour at the end of the sprint for filling out a Retro Template

**Comment by decause-gov** [comment]
06/16:
- Will focus on backend this week.
- stretchgoal: mockups if backend is solid.

**Comment by decause-gov** [comment]
06/21:
- This ticket depends on #97 in repo-scaffolder

**Comment by decause-gov** [comment]
06/24:
- Depends on reposcaffolder/#97, but, will update the frontend after the templates are fully baked, and mockups are complete.

**Comment by IsaacMilarky** [comment]
6/28:

- Made a couple mock-ups of the frontend Link: https://www.figma.com/design/wKwMcM5N00XYEU06viKvbT/CMS-OSPO-Metrics?node-id=90-124&t=XNaZJYAabT6epKjm-0
- ACTION: Get the mock-ups reviewed and create tickets from feedback.

**Comment by IsaacMilarky** [comment]
7/1:

- Progress made on existing figma mockups for the front end 
- ACTION: Finish mock-ups done
- STRETCH GOAL: Get working demo of front end with example data

**Comment by RicardoZamora01** [comment]
7/5:

- Mockups are done, would appreciate feedback!

**Comment by IsaacMilarky** [comment]
7/8:

- Mockup link: https://www.figma.com/design/wKwMcM5N00XYEU06viKvbT/CMS-OSPO-Metrics?node-id=90-124&t=lwkDcRG2ET69F89q-0
- Thinking about modifying tags 
- ACTION: Review mockups

**Comment by IsaacMilarky** [comment]
7/12:

- ACTION: @IsaacMilarky and @natalialuzuriaga give feedback in figma

**Comment by natalialuzuriaga** [comment]
7/15:
- Nat gave feedback on mockups
- @RicardoZamora01 will start on frontend for this - adding tags under title row

**Comment by natalialuzuriaga** [comment]
7/19:
- Frontend PR is up with 6 tags up! Needs to double check with @natalialuzuriaga on editing some options (repo host)

**Comment by decause-gov** [comment]
7/22:
- Fixing path issues in dev/main.
- Once merged, closing this ticket!
